src/main/java/me/clip/placeholderapi/util/TimeUtil.java

```diff
@@ -20,143 +20,60 @@
  */
 package me.clip.placeholderapi.util;
 
-public class TimeUtil {
+import org.bukkit.plugin.java.JavaPlugin;
 
-	public static String getRemaining(int seconds, TimeFormat type) {
-		
-		if (seconds < 60) {
-			switch(type) {
-			case DAYS:
-				return "0";
-			case HOURS:
-				return "0";
-			case MINUTES:
-				return "0";
-			case SECONDS:
-				return String.valueOf(seconds);
-			}
-			return String.valueOf(seconds);
-		}
-		
-		int minutes = seconds / 60;
-		int s = 60 * minutes;
-		int secondsLeft = seconds - s;
-		if (minutes < 60) {
-			switch(type) {
-			case DAYS:
-				return "0";
-			case HOURS:
-				return "0";
-			case MINUTES:
-				return String.valueOf(minutes);
-			case SECONDS:
-				return String.valueOf(secondsLeft);
-			}
-			return String.valueOf(seconds);
-		}
-		
-		if (minutes < 1440) {
-			int hours = minutes / 60;
-			int inMins = 60 * hours;
-			int leftOver = minutes - inMins;
-			switch(type) {
-			case DAYS:
-				return "0";
-			case HOURS:
-				return String.valueOf(hours);
-			case MINUTES:
-				return String.valueOf(leftOver);
-			case SECONDS:
-				return String.valueOf(secondsLeft);
-			}
-			return String.valueOf(seconds);
-		}
-		
-		int days = minutes / 1440;
-		int inMins = 1440 * days;
-		int leftOver = minutes - inMins;
-			
-		if (leftOver < 60) {
-			switch (type) {
-			case DAYS:
-				return String.valueOf(days);
-			case HOURS:
-				return String.valueOf(0);
-			case MINUTES:
-				return String.valueOf(leftOver);
-			case SECONDS:
-				return String.valueOf(secondsLeft);
-			}
-			return String.valueOf(seconds);
+import me.clip.placeholderapi.PlaceholderAPIPlugin;
+import me.clip.placeholderapi.configuration.PlaceholderAPIConfig;
+import me.clip.placeholderapi.util.TimeFormat;
 
-		} else {
-			int hours = leftOver / 60;
-			int hoursInMins = 60 * hours;
-			int minsLeft = leftOver - hoursInMins;
-			switch (type) {
-			case DAYS:
-				return String.valueOf(days);
-			case HOURS:
-				return String.valueOf(hours);
-			case MINUTES:
-				return String.valueOf(minsLeft);
-			case SECONDS:
-				return String.valueOf(secondsLeft);
-			}
-			return String.valueOf(seconds);
-		}	
-	}
+public class TimeUtil {
 	
-	public static String getTime(int seconds) {
+	public static final String getRemaining(int seconds, TimeFormat type) {
+		final PlaceholderAPIPlugin plugin = (PlaceholderAPIPlugin) JavaPlugin.getProvidingPlugin(PlaceholderAPIPlugin.class);
+		final PlaceholderAPIConfig config = plugin.getPlaceholderAPIConfig();
+		final String time = config.getTime(type);
+		final int days = seconds / 86400; seconds = seconds - (days * 86400);
```

Comment:
I would highly suggest taking advantage of the TimeUnits made specifically to do these sort of conversions. It would make the code a lot less verbose, and increase general readability in a function like that which tends to becomes a "wall" of code.